jest.config.js

```diff
@@ -1,3 +1,5 @@
+require('jest-preset-angular/ngcc-jest-processor');
```

Comment:
why is this only needed in this file but not in debug.config?

Reply:
Imported in debug.config.